src/components/DeploymentDisableToggle.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <p-tooltip text="Deployment enabled" side="left">
```

Comment:
Maybe we can make this a little more explicit with something like:
`Deployment ${enabled ? 'enabled' : 'disabled'} (click to ${enabled ? 'disable' : 'enable'})`, as-is it might be a little confusing